cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -534,3 +535,80 @@ For Kerberos-enabled Hadoop clusters:
     rm -rf /yarn/nm/usercache/cdap
 
   Restart CDAP after removing the usercache(s).
+
+.. _cloudera-configuration-highly-available:
+
+Enabling CDAP HA
+----------------
+In addition to having a :ref:`cluster architecture <admin-manual-install-deployment-architectures-ha>`
+that supports HA (high availability), these additional configuration steps need to followed and completed:
+
+CDAP Components
+...............
+For each of the CDAP components listed below (Master, Router, Kafka, UI, Authentication Server), these
+comments apply:
+
+- Sync the configuration files (such as ``cdap-site.xml`` and ``cdap-security.xml``) on all the nodes. 
+  Though the Cloudera Manager UI will do this for you, as any customization of individual can over-ride this, 
```

Comment:
and actually i'm not sure if this bullet is needed at all.  It's a little redundant to the next bullet, and i think it relates more to basic CM operations and may not be needed.  We can discuss...